### PR TITLE
fix: empty commit

### PR DESCRIPTION
**Pull request by k-huggs**
Empty commit to create version release
Updates semantic release config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +66,26 @@
...
     "plugins": [
-      "@semantic-release/commit-analyzer",
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "angular",
+          "releaseRules": [
+            {
+              "type": "chore",
+              "release": "minor"
```
Comment (kimon-satan) [inline package.json:76]: Sorry I think this should be patch

File: package.json [hunk 2]
```diff
@@ -66,7 +66,26 @@
...
+            },
+            {
+              "type": "feat",
+              "release": "patch"
```
Comment (kimon-satan) [inline package.json:80]: and this should be minor

**Review by kimon-satan — APPROVED** [review]
Sorry I told you it the wrong way around minor is reserved for feat. Patch is for the other two.

**Comment by k-huggs** [comment]
> Sorry I told you it the wrong way around minor is reserved for feat. Patch is for the other two.

Switched them around ✅

**Pull request merged**